test/memote.jl

```diff
@@ -46,308 +46,274 @@ end
-    ) == "MDH"
-
-    negative_model.reactions["FRD7"].ub = 0
-    negative_model.reactions["FRD7"].lb = -1000
-    crs = Network.find_cycle_reactions(negative_model, GLPK.Optimizer)
-    @test isempty(crs)
-
-    add_reaction!(negative_model, Reaction("temp", Dict("nad_c" => -1), :bidirectional))
-    d = Network.find_complete_medium_orphans_and_deadends(negative_model, GLPK.Optimizer)
-    @test length(d[:consume]) == 10 # able to consume nadp as well
-    @test length(d[:produce]) == 10 # able to produce nadp as well
+    @test !Biomass.biomass_missing_essential_precursors(iML1515, "BIOMASS_Ec_iML1515_core_75p37M")
 end
 
-@testset "Reactions" begin
-    ident_grrs = FBCModelTests.Memote.Reaction.reactions_with_identical_genes(model)
-    @test length(ident_grrs) == 11
-    @test issetequal(ident_grrs[["b1602", "b1603"]], ["NADTRHD", "THD2"])
-
-    metabolic_reactions_unconstrained, metabolic_reactions_constrained =
-        FBCModelTests.Memote.Reaction.find_all_purely_metabolic_reactions(model)
-    @test length(metabolic_reactions_unconstrained) == 50
-    @test length(metabolic_reactions_constrained) == 1
-
-    transport_unconstrained, transport_constrained =
-        FBCModelTests.Memote.Reaction.find_all_transport_reactions(model)
-    @test length(transport_unconstrained) == 23
-    @test length(transport_constrained) == 0
-
-    @test length(
-        FBCModelTests.Memote.Reaction.reactions_with_partially_identical_annotations(model),
-    ) == 14
-
-    @test issetequal(
-        FBCModelTests.Memote.Reaction.duplicate_reactions(model),
-        ["FRD7", "SUCDi"],
-    )
-
-    # negative tests
-    negative_model = deepcopy(convert(StandardModel, model))
-    negative_model.reactions["NADTRHD"].grr = [["b1602"]]
-    ident_grrs =
-        FBCModelTests.Memote.Reaction.reactions_with_identical_genes(negative_model)
-    @test length(ident_grrs) == 10
-
-    delete!(negative_model.reactions, "ATPM")
-    delete!(negative_model.reactions, "ICL")
-    metabolic_reactions_unconstrained, metabolic_reactions_constrained =
-        FBCModelTests.Memote.Reaction.find_all_purely_metabolic_reactions(negative_model)
-    @test length(metabolic_reactions_unconstrained) == 49
-    @test isempty(metabolic_reactions_constrained)
-
-    negative_model.reactions["FORt"].lb = -10
-    transport_unconstrained, transport_constrained =
-        FBCModelTests.Memote.Reaction.find_all_transport_reactions(negative_model)
-    @test length(transport_unconstrained) == 22
-    @test length(transport_constrained) == 1
-
-    negative_model.reactions["NH4t"].annotations = Dict("rhea" => ["1234"])
-    @test length(
-        FBCModelTests.Memote.Reaction.reactions_with_partially_identical_annotations(
-            negative_model,
-        ),
-    ) == 12
-
-    negative_model.reactions["SUCDi"].metabolites["succ_c"] = -2
-    @test isempty(FBCModelTests.Memote.Reaction.duplicate_reactions(negative_model))
-end
+# @testset "Consistency" begin
```

Comment:
I guess this code may be pwned completely as it is implicitly covered by the testcounting above, right?

Reply:
not sure what you mean. I have updated the tests now :)